src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/_transformers.py

```diff
@@ -105,6 +105,9 @@ def transform_url(result):
     """ Ensures the resulting URL string does not contain extra / characters """
     result = re.sub('//', '/', result)
     result = re.sub('/', '//', result, count=1)
-    return result
+    from six.moves.urllib.parse import quote, urlparse, urlunparse  # pylint: disable=import-error
```

Comment:
This import can really go at the top of file.
Also, what generates the import error?

Reply:
Not sure, but it seems to be a known problem:
https://github.com/PyCQA/pylint/issues/1640

Reply:
okay